recipe/meta.yaml

```diff
@@ -23,6 +25,7 @@ requirements:
     - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - make
+    - {{ mpi }}  # [mpi == 'openmpi' and build_platform != target_platform]
```

Comment:
ok!

Reply:
Just for reference, this is necessary due to https://github.com/conda-forge/scotch-feedstock/pull/61#issuecomment-1060980663 .